IFC4x3/Properties/u/Units_3hWrEPG6TBow7MJYMO7xCA/Documentation.md

```diff
@@ -0,0 +1 @@
+Specifies the units of measure used by this Linear Referencing Method for measures along the linear element being measured.
```

Comment:
Why "units" as plural?

Reply:
It is like that in the standard. Had the same thought myself...